dependencies/windows/install-dependencies.cmd

```diff
@@ -39,7 +39,13 @@ set PANDOC_VERSION=2.18
 set PANDOC_NAME=pandoc-%PANDOC_VERSION%
 set PANDOC_FILE=%PANDOC_NAME%-windows-x86_64.zip
 
-set QUARTO_VERSION=0.9.230
+REM set QUARTO_VERSION=0.9.230
```

Comment:
@timtmok The windows install-dependencies script is no longer working to install quarto for me with these changes. Can you take a look?

Reply:
I think the issue is I'm getting an unauthorized access from get-quarto-version.ps1

Reply:
```--2022-05-25 11:18:34--  https://github.com/quarto-dev/quarto-cli/releases/latest/download/quarto-
Resolving github.com (github.com)... 140.82.114.4
Connecting to github.com (github.com)|140.82.114.4|:443... connected.
HTTP request sent, awaiting response... 302 Found
Location: https://github.com/quarto-dev/quarto-cli/releases/download/v0.9.449/quarto- [following]
--2022-05-25 11:18:35--  https://github.com/quarto-dev/quarto-cli/releases/download/v0.9.449/quarto-
Reusing existing connection to github.com:443.
HTTP request sent, awaiting response... 404 Not Found
2022-05-25 11:18:35 ERROR 404: Not Found.

--2022-05-25 11:18:35--  http://+/
Resolving + (+)... failed: No such host is known. .
wget: unable to resolve host address '+'
--2022-05-25 11:18:35--  http://fullyqualifiederrorid/
Resolving fullyqualifiederrorid (fullyqualifiederrorid)... failed: No such host is known. .
wget: unable to resolve host address 'fullyqualifiederrorid'
:: Scheme missing.
--2022-05-25 11:18:36--  http://unauthorizedaccess-win.zip/
Resolving unauthorizedaccess-win.zip (unauthorizedaccess-win.zip)... failed: No such host is known. .
wget: unable to resolve host address 'unauthorizedaccess-win.zip'
Unzipping Quarto quarto-    + FullyQualifiedErrorId : UnauthorizedAccess-win.zip
unzip:  cannot find either ../quarto- or ../quarto-.zip.
Could Not Find C:\Users\jacquelinegutman\rstudio\dependencies\common\quarto-
```

Reply:
Working on my dev machine (Intel, Windows-11), and on the build machines which also run these dependency scripts.

Maybe related to ARM somehow (Jackie is using ARM), though not sure why that would matter here.

Reply:
I just tried this out on my new Amazon Workspaces windows machine which is Intel (AMD64, Windows Server 2019) and I get the same error I got from my ARM VM